packages/protocol/contracts/layer1/based/TaikoL1.sol

```diff
@@ -295,7 +295,7 @@ contract TaikoL1 is EssentialContract, ITaikoL1, TaikoEvents {
             blockRingBufferSize: 360_000, // = 7200 * 50
             maxBlocksToVerify: 16,
             blockMaxGasLimit: 240_000_000,
-            livenessBond: 125e18, // 125 Taiko token
+            livenessBond: 0.07 ether,
```

Comment:
any relevant comment about why this number is used

Reply:
Last point in PR description (checked with Lin from the research team):
> Bond amounts have been changed to ETH values based on current TAIKO price. Eg: Liveness bond is changed from 125 TAIKO to 0.07 ETH.